Prague 2020 Welcome Wagon update

bravegrape opened the pull request:
The Portland crew made some updates to the Welcome Wagon Google Doc.
This PR adds those updates to the Prague Welcome Wagon page.

bravegrape committed:
Add Welcome Wagon updates from Google doc

bravegrape commented on docs/conf/prague/2020/welcome-wagon.rst:
```diff
@@ -13,8 +13,7 @@ Write the Docs conference before.
 virtually meet you: feel free to reach out in Chat at the virtual
-conference if we can help
-make your time at the conference easier.
+conference if we can help make your time at the conference easier.
```
The Portland version also mentions email contact and links to @canncrochet 's email - which email should we use? 🤔

ddbeck replied:
During one of the planning calls, I think someone said that people weren't really reaching out to the Welcome Wagon specifically—sort of just throwing questions out there. It seems to me if nobody's going to email us, then we don't really need to provide an email

plaindocs replied:
That is definitely a self fulfilling prophecy though.

thatdocslady replied:
I think we can just use the portland@writethedocs.org email address, and we can route WW-specific emails to the right people as needed.

bravegrape replied:
I've added prague@writethedocs.org as the email contact 👍 

I also think it makes sense to have it there, in case people can't get in touch using the conference software - for example if they're having trouble joining or something similar.

bravegrape commented on docs/conf/prague/2020/welcome-wagon.rst:
```diff
@@ -58,6 +57,10 @@ Conference talks and Q&A
 - There’s a talk schedule on the **Reception** page and on the `Write the Docs website <https://www.writethedocs.org/conf/prague/2020/schedule/>`__. Select the Calendar icon next to the talks you want to see on the Reception page to add them to your calendar.
 - When it’s time for a talk you want to see, select **Stage**.
 - If we have problems with our online platform, go to the `Write the Docs website <https://www.writethedocs.org/conf/prague/2020/backup-livestream/>`__ for a back-up stream. This will only be turned on if we have technical issues.
```
This URL goes to Page not found - thanks for trying. It did make me chuckle, but should we avoid having a Page not found error?

We could have a page there saying the backup stream link will be shown here if we're having issues.
Or we could remove the hyperlink here and only add it in the emergency PR.

thatdocslady replied:
Good point, @mxsasha do you mind helping with this? We can just use some text that says "Here will be the live stream in case the conference platform blows up"...

bravegrape replied:
I've resolved this as discussed in https://github.com/writethedocs/www/pull/1345/commits/f12f085c2c0adda5464c837e2e08b0289cb9d2cd

bravegrape commented on docs/conf/prague/2020/welcome-wagon.rst:
```diff
@@ -58,6 +57,10 @@ Conference talks and Q&A
 - When it’s time for a talk you want to see, select **Stage**.
 - If we have problems with our online platform, go to the `Write the Docs website <https://www.writethedocs.org/conf/prague/2020/backup-livestream/>`__ for a back-up stream. This will only be turned on if we have technical issues.
+- To attend the moderated Q&A after each talk, go to **Sessions** and select the **Q&A session**.
+  - Enter your questions into the **Chat** channel (always on the right-hand side of any page) prefixed with “Q:“, and the moderator will ask the speaker for you.
```
@mxsasha just checking that this is what the script expects when it's looking for questions in the chat log 🔎 📄

bravegrape replied:
I'm resolving this one because we just tested it in the tech rehearsal 👍

bravegrape reviewed (commented):
@bethaitman and @ddbeck I've left some comments about the Welcome Wagon updates 😁 

Have a look and let me know what you think 👍 

I'm also wondering, does WTD have a style guide? I've spotted capitalisation inconsistencies in headings, plus we make liberal use of bulleted lists - what's our position on this?

bravegrape commented:
@ericholscher and @thatdocslady I think we agreed to rename the Job Fair to Career Clinic, right? Are we doing the find-and-replace in a different PR or should I put those changes in there?

I haven't copied the re-written Job Fair content from Portland because it seemed to assume a much higher number of participating companies. Could you help me figure out if/how the content needs changing for Prague 2020?

ddbeck commented on docs/conf/prague/2020/welcome-wagon.rst:
```diff
@@ -115,10 +118,15 @@ Unconference
 - The previous unconference session will be in the same table. Let the attendees of the previous session know that their time is up, and your session will begin soon.
-- Once a group has entered your session, start the conversation!
+- Once a group has entered your session, start the conversation! Here are some tips:
+  - Avoid doing roundtable introductions. You might run out of time to actually start the conversation!
```
This formatting isn't right. The nested list is [not a list at all in the preview](https://writethedocs-www--1345.org.readthedocs.build/conf/prague/2020/welcome-wagon/#to-lead-an-unconference-session).

bravegrape replied:
Oh no 🙈 
I did fix it but nested lists don't look good so I went back to having a separate subsection about asking questions during the Q&A session 👍

ddbeck replied:
It looks like you haven't pushed that change yet. I'm satisfied though, as long as it's fixed 😄

bravegrape replied:
I've pushed the changes in [this commit](https://github.com/writethedocs/www/pull/1345/commits/5e23a59a164cb0e7b3cd121d66e8495507b5d282) - and [the Netlify deployment looks ok to me](https://writethedocs-www--1345.org.readthedocs.build/conf/prague/2020/welcome-wagon/#conference-talks-and-q-a):

![image](https://user-images.githubusercontent.com/17963330/94557263-9ed6f100-025e-11eb-8543-4b7358a4ad79.png)

But it's true that I also didn't see the update if opening the build in a new tab. I can only see it if I stay in the same tab or go Incognito. Not sure what's going on here 🤔

thatdocslady commented:
> @ericholscher and @thatdocslady I think we agreed to rename the Job Fair to Career Clinic, right? Are we doing the find-and-replace in a different PR or should I put those changes in there?
> 
> I haven't copied the re-written Job Fair content from Portland because it seemed to assume a much higher number of participating companies. Could you help me figure out if/how the content needs changing for Prague 2020?

I think we're still waiting for Eric to finalize things with some of the sponsors. For now, let's keep the Job Fair as-is and if we end up converting it to something else, we can change it later :)

bravegrape committed:
Add contact email for Welcome Wagon page

bravegrape committed:
Keep Q&A info in a separate section

Having nested lists looks ugly. Keeping the info about asking questions 
in a separate subsection removes the need for a nested list and also 
improves readability.

bravegrape committed:
Remove unnecessary tabbed indentation for lists

bethaitman commented on docs/conf/prague/2020/welcome-wagon.rst:
```diff
@@ -13,8 +13,7 @@ Write the Docs conference before.
-make your time at the conference easier.
+conference if we can help make your time at the conference easier. You can also email us at {{ shortcode }}@writethedocs.org.
 
 .. contents:: All topics
```
Feel free to ignore this, but I thought the "all topics" looked a bit weird, and maybe that heading can just go - I don't think the TOC needs an introduction.

bravegrape replied:
I agree! Actually I completely missed the "All topics" 😂 
It's gone now!

bethaitman commented on docs/conf/prague/2020/welcome-wagon.rst:
```diff
@@ -193,27 +210,22 @@ You can join with your own audio and video, or chat only.
-  - When you chat on any of the main pages (Reception, Sessions, Networking, and Expo, you are chatting *the entire conference*.
-  - When you attend a Session, view the main Stage, or visit an Expo booth, you’ll see a toggle for switching between conference-wide chat and event chat. When you select **Event**, you are chatting the entire conference. To chat just your session, toggle on the **Session** or **Booth** chat.
 
 Pro Tips
```
maybe lowercase "t", seeing as the other titles are in sentence case?

bravegrape replied:
Switched to sentence case throughout 👌

bethaitman reviewed (approved):
Just some very tiny nitpicks.

The other thing I wasn't sure about was that in the test event, the schedule isn't on the Reception page. But maybe that's just for the test, or it'll get updated.

bravegrape committed:
Create separate section for unconference tips

Nested bulleted lists don't look good.

bravegrape committed:
Add info about the conference Slack channel

bravegrape committed:
Update what to do if there are technical issues

bravegrape committed:
Add email as a way of messaging organisers

bravegrape committed:
Style changes

bravegrape committed:
Remove "All topics" intro for the TOC

bravegrape committed:
Merge branch 'master' into ww-update-prague2020

bethaitman committed:
Merge branch 'master' into ww-update-prague2020

bravegrape committed:
Add URL to #wtd-conferences Slack channel
